ansible/build-gpu-oracle.yml

```diff
@@ -39,4 +39,7 @@
 # docker pull most recent running version of skynet and whisper
     - { role: "gpu-docker-pull", tags: "gpu-docker", gpu_docker_type: "skynet" }
     - { role: "gpu-docker-pull", tags: "gpu-docker", gpu_docker_type: "whisper" }
+    - { role: "streaming-whisper", tags: "gpu-docker", gpu_docker_type: "whisper" }
+    - { role: "autoscaler-sidecar", tags: "autoscaler-sidecar",
```

Comment:
This role is already included in the base image that GPU comes from, so no need to install it here.